Examples/Python/femur_cut.py

```diff
@@ -185,16 +185,16 @@ def Run_Pipeline(args):
             This function can handle both cases (processing only segmentation data or raw and segmentation data at the same time).
             This function uses the same transfrmation matrix for alignment of raw and segmentation files.
             """
-            [rigidFiles_segmentations, rigidFiles_images] = applyRigidAlignment(parentDir + "aligned", centerFiles_segmentations, centerFiles_images, medianFile, processRaw = True)
+            [resampledFiles_segmentations, resampledFiles_images] = applyResampling(parentDir + "resized", medianFile,centerFiles_segmentations, centerFiles_images)
```

Comment:
Why do we call the output of this step "resized"? This naming does not reflect the alignment step.

Reply:
The use cases are not doing rigid alignment right now, see issue #826. For now we are calling it "resized" because all it is doing in resampling is resizing the image if needed. I will tackle this rigid alignment issue next.